src/app/components/shell/header/header.scss

```diff
@@ -1,12 +1,30 @@
+// scss-lint:disable PropertySortOrder SelectorDepth IdSelector NestingDepth
```

Comment:
SCSS Lint rules shouldn't be disabled unless necessary.  If necessary, there should be a comment explaining why.